code/src/App.js

```diff
@@ -1,9 +1,171 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './components/Summary'
+import './app.css'
 
 export const App = () => {
+
+  //useState Hooks 
+  const [question, setQuestion] = useState(1)
+  const [text, setText] = useState("")
+  const [select, setSelect] = useState("")
+  const [radioButton, setRadioButton] = useState()
+  const [checkbox, setCheckbox] = useState()
+  const [range, setRange] = useState(5)
+  const [submitted, setSubmitted] = useState(false)   //False because form is not submitted from start (initial value)
+
+  //Array with values for mapping radio buttons in #3
+  const radioButtons = ["Apple", "Banana", "Minions?"]
+  //Array with values for mapping checkboxes in #4
+  const checkboxes = ["Yes", "No", "The slowest"]
+
+  //Nextbutton only enabled when text input is over 0 chars
+  const enabled = text.length > 0
+
+  //Function to handle next-button/onclick
+  const handleNext = () => {
+    setQuestion(question + 1)
+  }
+
+  //Function to handle next-button/onclick
+  const handleBack = () => {
+    setQuestion(question - 1)
+  }
```

Comment:
You can merge this into one function! So if you have one for handle the navigation, you can send `-1` or `+1` when you are clicking on the button. Then you can have a function like this:

```
const handleNavigation = (direction) => {
   setQuestion(direction)
}
```

Reply:
Will fix this!